packages/react/src/components/DataTable/TableToolbarSearch.js

```diff
@@ -55,8 +55,10 @@ const TableToolbarSearch = ({
     [searchContainerClass]: searchContainerClass,
     [`${prefix}--toolbar-action`]: true,
     [`${prefix}--toolbar-search-container-active`]: expanded,
-    [`${prefix}--toolbar-search-container-expandable`]: !persistant,
-    [`${prefix}--toolbar-search-container-persistant`]: persistant,
+    [`${prefix}--toolbar-search-container-expandable`]:
+      !persistent || !rest.persistant,
+    [`${prefix}--toolbar-search-container-persistent`]:
+      persistent || rest.persistent,
```

Comment:
Are `!rest.persistant` and `rest.persistent` here supposed to be the same? Either persistant or persistent

Reply:
yeah it should be `persistent` or `rest.persistant` in both. tough to intentionally spell incorrectly

Reply:
It really is, persistent stopped looking like a word after a while just reading this lol